src/selector.js

```diff
@@ -19,18 +19,30 @@ const isSelectableChoice = (file) => {
 export const fileSelector = createPrompt((config, done) => {
   const { basePath = "./", message, pageSize = 10, extensions = [".js", ".jsx", ".ts", ".tsx"] } = config;
 
-  const [cursorPosition, setCursorPos] = useState(1);
+  const [cursorPosition, setCursorPos] = useState(0);
```

Comment:
The initial index should be 1
```suggestion
  const [cursorPosition, setCursorPos] = useState(1);
```